mobile/src/components/course/course.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { WebView } from 'react-native-webview';
+
+import defaultCourseImage from '~/assets/images/default-course-image.png';
+import { DataStatus } from '~/common/enums/enums';
+import { Image, Spinner, Text, View } from '~/components/common/common';
+import { getImageUri } from '~/helpers/helpers';
+import { useAppSelector } from '~/hooks/hooks';
+
+import { styles } from './styles';
+
+const Course: FC = () => {
+  const { course, dataStatus } = useAppSelector((state) => state.courses);
+  const HTML = `
```

Comment:
Create a common HTMLContent component, you need to sanitize content in it and then pass it into WebView or you can also try this library react-native-render-html instead of web-view. You don't need to make a document from html content, just pass it as it is

Reply:
Now if I pass it as is, the text font is very small and I don't know how to change the font color then

Reply:
I will add screenshots

Reply:
We can pass styles as a string to webview, but let's use react-native-render-html, it looks more flexible and we can pass [tagsStyles](https://meliorence.github.io/react-native-render-html/docs/guides/styling#mixed-style-records) to it as an object

Reply:
Also pay attention that we should scroll all the screen not only description

Reply:
if i pass just description without meta tag some pages look like this

<img src="https://i.gyazo.com/5dcd764c42742d66f8e22192d877dd79.png" width="100%">

Reply:
I tried use sanitizeHTML like: `const sanitizedHTML = sanitizeHTML(html);` but i got error

<img src="https://i.gyazo.com/e5b269e3a94e1f96a6c71e2b85ce1228.png" width="40%">

Reply:
i will try to replace webview package because it doesn't work with scrollview if i use scrollview and webview together then i need to add fixed height of webview but i can't do that because content size is different

Reply:
@Nailheart lib from shared is not supported for `react-native`, so please use `sanitize-html`. Create a helper into `mobile` folder and import `course` component